addons/website/tests/test_assets.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+
+from odoo.tools import config
+
+
+@odoo.tests.common.tagged('post_install', '-at_install')
+class TestWebsiteAssets(odoo.tests.HttpCase):
+
+    def test_01_multi_domain_assets_generation(self):
+        Website = self.env['website']
+        Attachment = self.env['ir.attachment']
+        # Simulate single website DBs: make sure other website do not interfer
+        # (We can't delete those, constraint will most likely be raised)
+        Website.search([]).write({'domain': 'inactive.test'})
+        # Don't use HOST, hardcode it so it doesn't get changed one day and make
+        # the test useless
+        domain_127_0_0_1 = "http://127.0.0.1:%s" % config['http_port']
+        # domain_127_0_0_2 = "http://127.0.0.2:%s" % config['http_port']
```

Comment:
runbot is not happy with this so I'll use `localhost` and `127.0.0.1` I guess, rather than `127.0.0.1` and `127.0.0.2`.
Test still break without the fix anyway
![image](https://user-images.githubusercontent.com/30048408/231816403-d3e82844-b76b-4282-aac2-4d167df21263.png)